### PR TITLE
Cleaner view jesica

### PR DESCRIPTION
**Pull request by jesicaj99**
Styled index page for all except admin_request
Styled Edit, Show, and New files for Ranking
Ranking updates for all users in system improved

### CODE REVIEW

File: app/controllers/accounts_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class AccountsController < ApplicationController
   def index
+
```
Comment (caleb-terry17) [inline app/controllers/accounts_controller.rb:3]: should this empty line be removed?

File: app/views/rankings/show.html.erb [hunk 2]
```diff
@@ -1,14 +1,37 @@
+<h1>Show Ranking</h1>
+
```
Comment (caleb-terry17) [inline app/views/rankings/show.html.erb:2]: spacing of lines in this file does not seem to be consistent with the rest of the application.

**Review by caleb-terry17 — APPROVED** [review]
- updated gemfile 
- made styling changes to the application pages
- added styling for buttons 
- added styling for events
- added styling for forms
- added styling for "show" pages
- added styling for the table
- made updates to ranking users
- updated styling on attendances pages
- updated styling on event_types pages
- updated styling on events pages
- updated styling on history pages
- added welcome statement to landing page
- 
- follows coding standards

**Review by amaredia3 — APPROVED** [review]
Simplecov files have errors but we can fix them later.

**Pull request merged**